### PR TITLE
add spread stateful function

### PR DESCRIPTION
**Pull request by jsvisa**

### CODE REVIEW

File: tick/functions.go [hunk 1]
```diff
@@ -397,6 +398,37 @@ func (s *sigma) Call(args ...interface{}) (interface{}, error) {
...
+	min float64
+	max float64
+}
+
+func (s *spread) Reset() {
+	s.min = math.Inf(+1)
+	s.max = math.Inf(-1)
+}
+
+// Computes the number of a given value is from the running mean.
```
Comment (nathanielc) [inline tick/functions.go]: Can you update this comment?

**Comment by nathanielc** [comment]
@jsvisa Looks good. Before I can merge can you sign the [CLA](http://influxdb.com/community/cla.html) and update the comment? 

Thanks!

**Comment by jsvisa** [comment]
@nathanielc Sorry for the comment, just updated right now. I've signed the [CLA | InfluxDB](https://influxdb.com/community/cla.html).

**Comment by nathanielc** [comment]
@jsvisa Thanks, can you rebase your changes at this point?

**Comment by jsvisa** [comment]
@nathanielc rebased.

**Comment by nathanielc** [comment]
@jsvisa Looks good, can you add a changelog entry and I'll merge? Thanks!

**Comment by jsvisa** [comment]
@nathanielc I added a new entry inside v0.14.0's features, is it suitable?

**Comment by nathanielc** [comment]
@jsvisa Yep, that's good. Can you rebase one more time so I can merge? I'll merge on green.

**Pull request merged**